kubejs/server_scripts/base/tags/items/forge/cobblestone.js

```diff
@@ -0,0 +1,5 @@
+onEvent('item.tags', (event) => {
+    cobblestones.forEach((cobblestone) => {
+        event.add("forge:cobblestone", cobblestone);
```

Comment:
This event accepts arrays, so you can just say `event.add("forge:cobblestone", cobblestones)`